src/core/main.cpp

```diff
@@ -510,6 +510,15 @@ int main( int argc, char * * argv )
 		Engine::getSong()->loadProject( file_to_load );
 		printf( "done\n" );
 
+		if ( Engine::getSong()-> hasErrors() )
+		{
+			const char * summary = Engine::getSong()->errorSummary()->toStdString().c_str();
+
+			fputs( "\n", stderr );
+			fputs( summary, stderr );
+			fputs( "\n", stderr );
```

Comment:
Also, following your advice see [qDebug documentation](http://qt-project.org/doc/qt-4.8/qdebug.html). This is clearly not what I want to do.

`fputf` already does what I need and noone is suggesting an alternative.